sygnal/notifications.py

```diff
@@ -100,19 +101,19 @@ def __init__(self, name: str, sygnal: "Sygnal", config: Dict[str, Any]):
         self.cfg = config
         self.sygnal = sygnal
 
-    def get_config(self, key: str, default=None):
+    def get_config(self, key: str, default=None) -> Optional[Union[str, int, Dict]]:
         if key not in self.cfg:
             return default
         return self.cfg[key]
 
     async def dispatch_notification(
         self, n: Notification, device: Device, context: "NotificationContext"
-    ) -> List[str]:
+    ) -> Optional[List[str]]:
```

Comment:
I think this should be backed out; the list of pushkeys should always be returned.

Probably replace `pass` below with `raise NotImplementedError()` as I guess Mypy will be sad that you're not returning anything

Reply:
Or maybe better:
I didn't know about this back in the day when I did the Sygnal → Python 3 port, but we could properly mark `Pushkin` as an Abstract Base Class (unhelpfully shortened to `ABC`).
You can then annotate  `@abstractmethod` on this method https://docs.python.org/3/library/abc.html#abc.abstractmethod
and replace the body with `...` (that's right: `...` is a valid expression in Python (crazy eh? I thought so at least!))

`class Pushkin:` → `class Pushkin(abc.ABC):`